tests/test_utils/test_nan_warnings.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test the RadViz feature analysis visualizers
+"""
+
+import unittest
+import numpy as np
+
+from yellowbrick.utils.nan_warnings import warn_if_nans_exist, count_nan_rows, \
+    drop_rows_containing_nans
+
+
+class TestNanWarnings(unittest.TestCase):
+    def test_raise_warning_if_nans_exist(self):
+        """
+        Test that a warning is raised if any nans are in the data
+        """
+        data = np.array([
+            [1, 2, 3],
+            [1, 2, np.nan],
+        ])
+
+        self.assertWarns(UserWarning, warn_if_nans_exist, data)
```

Comment:
We're actually in the process of switching over to Pytest - would prefer `pytest.warns` ([assert warnings with pytest](https://docs.pytest.org/en/latest/warnings.html#warns))  -- it'll be one less thing we have to do as we move towards a complete change over!